### PR TITLE
[dev] [No Bug/PBI] Remove console warnings about use of `disabled` prop

### PR DESCRIPTION
**Pull request by groberts314**
* We want to reverse ourselves, and favor `disabled` as the standard prop for disabling an interactive element (versus `disable` with no ending 'd').  Reasons: this is the standard in use for the native DOM and for MUI and such.
* We will continue to support `disable` as well on components that got adapted for that during the era where we were favoring that, but the documentation has been updated to indicate `disable` is deprecated and `disabled` is favored.  Also all of the console warnings have been removed.

See:
https://github.com/saddlebackdev/church-management/pull/5935#discussion_r804643925

### CODE REVIEW

**Review by Frankgcb — APPROVED** [review]
👍

File: docs/src/inputs/button/api/api.jsx [hunk 1]
```diff
@@ -4,9 +4,9 @@ import {
 import React from 'react';
 import ComponentApi from '../../../global/componentApi';
 import Main from '../../../global/main';
-/* eslint-disable import/no-named-default, import/extensions */
-import { default as rootDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/inputs/button/button';
-/* eslint-enable import/no-named-default, import/extensions */
+/* eslint-disable import/no-named-default, import/extensions, import/no-unresolved */
+import { default as rootDoc } from '!!@advclb/react-docgen-loader!react-cm-ui/inputs/button/button'; // eslint
```
Comment (groberts314) [inline docs/src/inputs/button/api/api.jsx:8]: Do we need `// eslint` here?

File: src/inputs/checkbox/checkbox.jsx [hunk 2]
```diff
@@ -11,14 +11,23 @@ const propTypes = {
...
      */
     disabled: PropTypes.bool,
     fluid: PropTypes.bool,
+    /**
+     * Forwarded Ref
+     */
+    forwardedRef: PropTypes.oneOfType([
+        // Either a function
+        PropTypes.func,
+        // Or the instance of a DOM native element (see the note about SSR)
```
Comment (groberts314) [inline src/inputs/checkbox/checkbox.jsx:28]: Did I put `(see the note about SSR)`?  I wonder why?  I must have copied that from somewhere.  I have _NO IDEA_ what it means! =)

File: src/inputs/dropdownButton/dropdownButton.jsx [hunk 3]
```diff
@@ -16,18 +19,66 @@ import Icon from '../../dataDisplay/icon';
...
+     */
+    fluid: PropTypes.bool,
+    /**
+     * The Dropown Button will be resized to its parent container's width. (Using v2 Button)
+     */
+    fullWidth: PropTypes.bool,
+    /**
+     * ??? I don't know what to put here because I don't fully understand what this does =)
+     */
+    icon: PropTypes.bool,
```
Comment (groberts314) [inline src/inputs/dropdownButton/dropdownButton.jsx:78]: Señor @cbrewer1689 ... could you help a brother out here? ;-)

**Review by groberts314 — COMMENTED** [review]
Looking good.  Juts a few things.

File: docs/src/inputs/button/api/api.jsx (hunk 1 above)
Comment (cbrewer1689) [reply docs/src/inputs/button/api/api.jsx]: Fixed

File: src/inputs/checkbox/checkbox.jsx (hunk 2 above)
Comment (groberts314) [reply src/inputs/checkbox/checkbox.jsx:28]: Yeah, if you would be so kind as to remove all of these, that'd be great.  I think it's clear without that.  It's either a function (like a component) or it's a native DOM element.  Period.  Thanks!  And sorry about that.

Comment (groberts314) [reply src/inputs/checkbox/checkbox.jsx:28]: OOOH ... I think SSR = "server-side rendering"!
Which, we don't do, so it's not applicable.

File: docs/src/inputs/button/api/api.jsx (hunk 1 above)
Comment (groberts314) [reply docs/src/inputs/button/api/api.jsx]: Looks good.  Resolving.

File: src/inputs/dropdownButton/dropdownButton.jsx (hunk 3 above)
Comment (cbrewer1689) [reply src/inputs/dropdownButton/dropdownButton.jsx]: Fixed

Comment (groberts314) [reply src/inputs/dropdownButton/dropdownButton.jsx]: Lookin' good.  Thanks!  https://github.com/saddlebackdev/react-cm-ui/pull/362/commits/894ed34061c181a2ef1103ca6bd6320635911475#diff-484bd1d32d50baad395b1f4b4222738641cbac745957095adb3859a0bb833788L76-R76

**Pull request merged**